app/models/mhv_account.rb

```diff
@@ -72,6 +80,10 @@ def preexisting_account?
     user&.mhv_correlation_id.present?
```

Comment:
This seems like it needs changing to reflect addition of mhv_uuid.

Reply:
Yea, so mhv_correlation_id now is basically:

`mhv_uuid || mvi.va_profile.mhv_correlation_id` on user.rb

where mhv_uuid has precedence, there is another method I created `mhv_signin?` which tests/checks for this.